File: PrimeNum.cpp
#include<bits/stdc++.h>
int main(){
    int a,i,count=0;
    std::cout <<"Enter a number :";
    std::cin >>a;
    for(i=a;i>0;i--){
        if(a%i==0){
            count+=1;
        }
    }
    if(count==2){
        std::cout << "The number is prime .";
    }
    else{
        std::cout << "The number is not prime .";
    }
}

File: PrimeNum.java
public class PrimeNum{
    public static void main(String[] args){
            int i,j,count=0;
            System.out.println("Prime numbers from 1 to 10 are :");
            for(j=1;j<11;j++){
                count=0;
                for(i=j;i>0;i--){
                    if(i%j==0){
                        count+=1;
                    }
                }
                if(count==2){
                    System.out.println(j);
                }
            }
    } 
}
